surf-for-type-safety/frontend/components/NetworkInfo.tsx

```diff
@@ -0,0 +1,42 @@
+import { Network } from "@aptos-labs/ts-sdk";
+import { useWallet } from "@aptos-labs/wallet-adapter-react";
+// Internal components
+import { DisplayValue, LabelValueGrid } from "@/components/LabelValueGrid";
+import { isValidNetworkName } from "@/utils/helpers";
+
+export function NetworkInfo() {
```

Comment:
we can easily remove the wallet info components - not relevant for demonstrating Surf usage